Jenkinsfile_develop

```diff
@@ -1,10 +1,10 @@
-//Jenkins pipelines are stored in shared libaries. Please see: https://github.com/tijcolem/nrel_cbci_jenkins_libs 
- 
-@Library('cbci_shared_libs') _
+//Jenkins pipelines are stored in shared libaries. Please see: https://github.com/tijcolem/nrel_cbci_jenkins_libs
 
-// Build for PR to develop branch only. 
+@Library('cbci_shared_libs@Clean_tests_for_CI') _
```

Comment:
Revert to using develop after https://github.com/NREL/cbci_jenkins_libs/pull/75 is merged

Reply:
Merged and update ref in this branch. https://github.com/NREL/cbci_jenkins_libs/pull/75